cfme/tests/infrastructure/test_scvmm_specific.py

```diff
@@ -30,6 +47,74 @@ def vm(provider, small_template):
     vm.cleanup_on_provider()
 
 
+@pytest.fixture
+def cfme_vhd(provider, appliance):
+    """ Given a stream from the appliance, gets the cfme vhd"""
+    stream = appliance.version.stream()
+    try:
+        url = '{}/'.format(conf.cfme_data["basic_info"]["cfme_images_url"][stream])
+    except KeyError:
+        pytest.skip("No such stream: {} found in cfme_data.yaml".format(stream))
+    # get image name
+    image_name = get_vhd_name(url)
+    if not image_name:
+        pytest.skip("No hyperv vhd image at {}".format(url))
+    # download the image to SCVMM library
+    provider.mgmt.download_file(url + image_name, image_name)
+    provider.mgmt.update_scvmm_library()
```

Comment:
Q: what do you think about adding some check that the image was successfully downloaded and added to the library?

Reply:
Done, depends on https://github.com/ManageIQ/wrapanapi/pull/384, waiting for new wrapanapi release then I'll move from WIP to RFR